core-services/image-mirroring/integr8ly/mapping_integr8ly_operator_release-rhoam-v1_0

```diff
@@ -0,0 +1,2 @@
+registry.svc.ci.openshift.org/integr8ly/release-rhoam-v1.0:integreatly-operator quay.io/integreatly/integreatly-operator:release-rhoam-v1.0
```

Comment:
@MStokluska I know you were involved in some image mirror conversations around the pipelines along with @philbrookes are these the mappings you would expect for the branch?

Reply:
would it be better to mirror it to https://quay.io/integreatly/managed-api-service:rhoam-v1.0.0 - I am not sure about the tag, but following with what we have so far in that repo.

Reply:
yes, it should match the release tag in rhoam, think this is probably a fix we need to do in delorean. For now do we want to have the image mirroring override the one in quay or are we creating that manually when we are cutting the rcs?